test/unit/legacy_wrappers/db.test.js

```diff
@@ -0,0 +1,284 @@
+'use strict';
+
+const {
+  MongoClient,
+  ListCollectionsCursor,
+  ChangeStream,
+  AggregationCursor,
+  Collection
+} = require('../../../src/index');
+const mongodbDriver = require('mongodb');
+const { Db } = require('mongodb');
+const sinon = require('sinon');
+const { expect } = require('chai');
+
+describe('legacy_wrappers/db.js', () => {
+  let db;
+  let client;
+  beforeEach(async function () {
+    client = new MongoClient('mongodb://iLoveJs');
+    db = client.db();
+  });
+
+  afterEach(async function () {
+    sinon.restore();
+    await client.close();
+  });
+
+  it('should return legacy listCollections cursor', () => {
+    expect(db.listCollections()).to.be.instanceOf(ListCollectionsCursor);
+  });
+
+  it('should return legacy ChangeStream', () => {
```

Comment:
can we differentiate between this expect and the one in the it below?  and can we specify the actual methods called both above, in this line, and in line below, for clarity of which method is being tested?